# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/.gitignore
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/.gitignore b/third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/.gitignore
deleted file mode 100644
--- a/third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/.gitignore
+++ /dev/null
@@ -1,20 +0,0 @@
-*.o
-*.d
-*.txt
-/test
-/spirv-cross
-/obj
-/msvc/x64
-/msvc/Debug
-/msvc/Release
-*.suo
-*.sdf
-*.opensdf
-*.shader
-*.a
-*.bc
-/external
-.vs/
-*.vcxproj.user
-
-!CMakeLists.txt